From 01f9a19da0aa7c6499c1ee9436659b4d128cebb2 Mon Sep 17 00:00:00 2001
From: Alexey Sokolov <sokolov@google.com>
Date: Tue, 26 Oct 2021 22:46:15 +0100
Subject: [PATCH] Fix build with clang

---
 src/common/Sockets.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/Sockets.cpp b/src/common/Sockets.cpp
index dee5643..1452c6c 100644
--- a/src/common/Sockets.cpp
+++ b/src/common/Sockets.cpp
@@ -143,7 +143,7 @@ DWORD CASC_SOCKET::GetAddrInfoWrapper(const char * hostName, unsigned portNum, P
                 continue;
             }
 #endif
-            case EAI_AGAIN:             // Temporary error, try again
+            case (DWORD)EAI_AGAIN:             // Temporary error, try again
                 continue;
 
             default:                    // Any other result, incl. ERROR_SUCCESS
